client/post-editor/editor-action-bar/style.scss

```diff
@@ -13,6 +13,18 @@
 	}
 }
 
+.editor-action-bar__cell {
+	width: 33%;
```

Comment:
I stumbled upon this StackOverflow question which appears to pose the same problem we're encountering:

http://stackoverflow.com/questions/32378953/how-can-you-keep-the-center-box-centered-in-a-flex-box-layout

I actually quite like [one of the answers](http://stackoverflow.com/a/37203728) which advocates applying 50% width to left and right cells with `flex-shrink: 1`, then `flex-shrink: 0` to the center column. This way the left and right columns can grow as much as needed while ensuring that the center is kept centered and never collapses.